evals/fixtures/dynamic-code-execution.ts

```
@@ -0,0 +1,40 @@
+const actionHandlers = {
+  refreshToken,
+  poll,
+} satisfies Record<ActionName, () => void>;
+
+export function runConfiguredAction(action: string): void {
+  if (action in actionHandlers) {
+    actionHandlers[action as ActionName]();
+    return;
+  }
+
+  eval(action);
```

Biome (2.5.1)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)